### PR TITLE
Hotfix/bmauer/fixes #1269 off main

### LINKED ISSUE #1269: Adding attributes that were unintentionally eliminted when History moved from cfio to PFIO
It was pointed out that during the transition from the History Component using cfio to pfio some attributes both at the global and variable level were lost. In addition the coordinate variables changed from double to single precision.
I.E.
```
        float PS(time, nf, Ydim, Xdim) ;
                PS:long_name = "surface_pressure" ;
                PS:units = "Pa" ;
                PS:_FillValue = 1.e+15f ;
                PS:missing_value = 1.e+15f ;
                PS:fmissing_value = 1.e+15f ;
                PS:scale_factor = 1.f ;
                PS:add_offset = 0.f ;
                PS:standard_name = "surface_pressure" ;
                PS:vmax = 1.e+15f ;
                PS:vmin = -1.e+15f ;
                PS:valid_range = -1.e+15f, 1.e+15f ;
```
and globally
```
// global attributes:
                :Title = "3d,1-Hourly,Instantaneous,Native Cubed-Sphere,Model-Level,Assimilation,Diagnostics" ;
                :_NCProperties = "version=2,netcdf=4.7.0,hdf5=1.8.21," ;
                :History = "File written by MAPL_CFIO" ;
                :Source = "GEOSadas-5_27_1_p7_OPS" ;
                :Contact = "http://gmao.gsfc.nasa.gov" ;
                :Conventions = "CF" ;
                :Institution = "NASA Global Modeling and Assimilation Office" ;
                :References = "see MAPL documentation" ;
                :Filename = "f5271_fp.inst_1hr_ctm_c0720_v72.20220105_0900z.nc4" ;
                :Comment = "NetCDF-4" ;
```
The fmissing_value,missing_value,scale_factor,add_offset,standard_name were eliminated as quite frankly these were not used, unnecessary or don't make sense with the values used. However, apparently scripting used internally and externally by customers may relying on this so we need to reintroduce it and then slowly remove it after consultation.

### PR DESCRIPTION
**Pull request by bena-nasa**
This fixes issue #1269 and updates the file metadata to match the old format as defined in the GEOS file spec as far as metadata in the file is concerned other than time. I have kept time as a float in this PR as that is what is really should be.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Trivial change (affects only documentation or cleanup)

## Checklist:

- [X] I have tested this change with a run of GEOSgcm (if non-trivial)
- [X] I have added one of the required labels (0 diff, 0 diff trivial, 0 diff structural, non 0-diff)
- [X] I have updated the CHANGELOG.md accordingly following the style of [Keep a Changelog](https://keepachangelog.com/en/1.0.0/#how)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Changes to make MAPL 2 History output match GEOS FP file spec
```
Comment (tclune) [inline CHANGELOG.md:16]: @mathomp4 - I wonder if this should be in the ### Fixed section instead?

**Review by tclune — COMMENTED** [review]
Looks fine modulo the one question about the CHANGELOG

File: CHANGELOG.md (hunk 1 above)
Comment (bena-nasa) [reply CHANGELOG.md:16]: I think changed is appropriate. I mean, nothing was "broken" in that it was perfectly fine code.

Comment (tclune) [reply CHANGELOG.md:16]: I won't debate the issue for the sake of urgency.  But "fixed" is for anything where the result is not as intended.   Relatively harmless in this case, but still incorrect from an ops consistency point-of-view.   "changed" is more for intentional evolution of features.  But as long as it is in the changelog the philosophers can debate.

**Review by tclune — APPROVED** [review]

**Pull request merged**